.github/workflows/cleanEnvironments.js

```diff
@@ -0,0 +1,113 @@
+const { Command } = require("commander");
+const fetch = require("node-fetch");
+
+const program = new Command();
+
+program
+  .name("cleanEnvironments")
+  .description("Clean environments")
+  .option("--token <token>", "token fo login to cloud")
+  .option("--snapshot <snapshot>", "snapshot to revert to")
+  .action(async options => {
+    const token = options.token;
+    const snapshot = options.snapshot;
+    const tasksArray = [];
+    const environmentsToClean = await getEnvironmentsForReleaseTesting(token);
+    environmentsToClean.forEach(environment => {
+      tasksArray.push(cleanEnvironment(environment, snapshot, token));
+    });
+    const throwErrorAfterTimeout = setTimeout(function () {
+      throw new Error("Environment didn't upgraded after 30 minutes");
+    }, 1800000);
+    Promise.all(tasksArray).then(tasksList => {
+      tasksList.forEach(task => {
+        if (task.error) {
+          console.warn(`${task.environment}: ${task.error}`);
```

Comment:
isn't that something we want action to fail?

Reply:
also it felt that's something belonging to `cleanEnvironment` function.

Reply:
I don't want to the whole workflow fail, eg. if cleaning the environment with version 3.9 fail because of some reason I still want to try to clean up environments 3.10, 3.11, 3.12...